policy/lib/rule_data.rego

```diff
@@ -0,0 +1,38 @@
+package lib
+
+#
+# Will define built-in default values here.
+# They can be overridden if required.
+#
+rule_data_defaults := {
+	#
+	# Used in release/attestation_type
+	"known_attestation_types": ["https://in-toto.io/Statement/v0.1"],
+	#
+	# Used in release/slsa_provenance_available
+	"allowed_predicate_types": ["https://slsa.dev/provenance/v0.2"],
+	#
+	# Used in release/slsa_build_build_service
+	"allowed_builder_ids": ["https://tekton.dev/chains/v2"],
+	#
+	# For the tests only. Not sure how else to get to 100% coverage.
+	"_test_key_3": 10,
```

Comment:
You could have a test that uses `rule_data_defaults` as is, and on that overwrites it by using `with`.

Reply:
Let me try...

Reply:
Works good. Thanks. I forgot you can use with for other stuff.